.github/workflows/template-only-cd.yml

```diff
@@ -10,9 +10,27 @@ on:
 concurrency: platform-template-only-cd
 
 jobs:
+  # TODO: Get rid of this job once we've merged the platform-cli branch into the main branch
+  update-platform-cli-branch:
+    name: Update platform-cli branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout template-infra repo
+        uses: actions/checkout@v4
+        with:
+          path: template-infra
+          ref: lorenyu/platform-cli
```

Comment:
This is intentional, right? Because the platform CLI PR hasn't been merged into main yet?

Reply:
That's correct

Reply:
We're gonna let it run in a branch for a little while first